src/homework/db/crud.py

```diff
@@ -78,3 +78,23 @@ def delete_application(app_id: str) -> bool:
     except (exc.SQLAlchemyError, exc.DBAPIError):
         return False
     return True
+
+
+def get_user_balance_by_id(user_id: int) -> None | int:
+    """
+    Get the user's balance from the database by their id.
+
+    If the user with given user ID does not exist, function returns None.
+    :param user_id: int
+    :return: None | int
+    """
+    with Session(engine) as session:
+        try:
+            balance = session.scalar(
+                select(User.current_balance).where(User.user_id == user_id)
+            )
+            if balance is None:
```

Comment:
I suggest deleting this check for None and returning balance object as is. Anyway it is either None, or int.
```python
if balance is None:
    return None
return balance
```
to 
```python
return balance
```

Reply:
Yes, thank you for this remark.